tools/devops/automation/templates/main-stage.yml

```diff
@@ -151,11 +151,18 @@ parameters:
 - name: azureContainer
   type: string
 
+- name: stageDisplayNamePrefix
+  type: string
+  default: ''
+
```

Comment:
Can you do something like this:

```yaml
variables:
    - name: stageDisplayNamePrefixWithInfix
${{ if ne(parameters.stageDisplayNamePrefix, '') }}:
    - value: ""
${{ else }}:
    - value: "${{ parameters.stageDisplayNamePrefix }} - "
 ```

and then later:

```yaml
    displayName: '$(stageDisplayNamePrefixWithInfix)Governance Checks'
```

(this is pseudo-yaml-code-that-will-never-work-as-is, but it should get the idea across)

This would avoid a lot of code duplication in this PR.

Reply:
meh, fair enough